### PR TITLE
WV-1943 DAAC Facet

### PR DESCRIPTION
**Pull request by jasontk19**
## Description

- Add facet for DAAC/SIPS to layer picks

## How To Test

- Open layer picker
- Switch to search/filter mode
- Try filtering by DAAC/SIPS facet

@nasa-gibs/worldview

### CODE REVIEW

**Comment by jasontk19** [comment]
Will need to wait until related GIBS PR is merged and in PROD to properly test

**Review by minniewong — COMMENTED** [review]
The DAAC/SIPS tooltip doesn't seem to work

File: tasks/python3/getVisMetadata.py
Comment (Benjaki2) [inline tasks/python3/getVisMetadata.py:68]: @jasontk19 I feel like we may want to remove some of these prints. I am getting a printout of a massive object and I think this could make it harder to debug other issues.

**Review by Benjaki2 — CHANGES_REQUESTED** [review]

File: tasks/python3/getVisMetadata.py
Comment (jasontk19) [reply tasks/python3/getVisMetadata.py:68]: Whoops yeah that shouldn't still be in there, good catch

**Comment by jasontk19** [comment]
> The DAAC/SIPS tooltip doesn't seem to work

How important do you think the links are for this?  We tried to get hyperlinks working in the tooltips before and went back and forth a bunch on how to get it to work satisfactorily.  I think we tried to get the tooltips to stay open longer before auto-hiding but that made the other tooltips feel sluggish when you don't need them to stay open in order to click a link.  We could probably do some more work to get some tooltips to stay open and some to auto-hide when you move the mouse away but that feels like work for a separate ticket.  If we don't absolutely need them for this I can just remove the links for now.

**Comment by minniewong** [comment]
> > The DAAC/SIPS tooltip doesn't seem to work
> 
> How important do you think the links are for this? We tried to get hyperlinks working in the tooltips before and went back and forth a bunch on how to get it to work satisfactorily. I think we tried to get the tooltips to stay open longer before auto-hiding but that made the other tooltips feel sluggish when you don't need them to stay open in order to click a link. We could probably do some more work to get some tooltips to stay open and some to auto-hide when you move the mouse away but that feels like work for a separate ticket. If we don't absolutely need them for this I can just remove the links for now.

Sounds good - let's just get the tooltip to spell out what DAAC and SIPS means for now, and we can open a separate ticket for hyperlinks in the tooltips.

File: web/js/modules/product-picker/facet-config.js
Comment (minniewong) [inline web/js/modules/product-picker/facet-config.js:61]: change it to singular "Center"

**Review by minniewong — APPROVED** [review]
🎸

**Review by Benjaki2 — APPROVED** [review]
👍

**Pull request merged**
